xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1093,6 +1091,27 @@ Status IrEmitterUnnested::EmitFusedMHAThunk(mlir::Operation* op) {
                           GetAllocationSlice(fmha.getActivation()));
     }
 
+    if (fmha.getBias() != nullptr) {
+      descriptor.bias_shape = ShapeUtil::MakeShapeWithDenseLayout(
+          GetShape(fmha.getBias()).element_type(),
+          GetShape(fmha.getBias()).dimensions(),
+          GetShape(fmha.getBias())
+              .layout()
+              .minor_to_major());
+
+      TF_ASSIGN_OR_RETURN(
+          bias_slice, GetAllocationSlice(fmha.getBias()));
+    }
+
+    if (fmha.getMask() != nullptr) {
```

Comment:
nit: I think `if (fmha.getMask())` is more idiomatic when working with MLIR `Value`s? (same for others?)